examples/sentencepiece/example_bpm_dataset.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import torch
+import io
+from torchtext.utils import download_from_url, extract_archive, unicode_csv_reader
+from torchtext.data.utils import generate_sp_tokenizer
+from os import path
+
+URLS = {
```

Comment:
You can get this from https://github.com/pytorch/text/blob/master/torchtext/datasets/text_classification.py#L11

Reply:
This file is a similar version of `text_classification`, should some structures be reused, or copied (as done here) appropriate?